Automattic/jetpack
# Duplicate views: Hide "View" switcher

Fixes https://github.com/Automattic/wp-calypso/issues/97395

## Proposed changes:

Hides the "View" switcher on any WP Admin screen that is enforced because of the remove duplicate views experiment.

Before | After
--- | ---
<img width="309" alt="Screenshot 2024-12-12 at 11 27 54" src="https://github.com/user-attachments/assets/f87f8036-16a5-4862-a65b-56300972018a" /> | <img width="322" alt="Screenshot 2024-12-12 at 14 54 51" src="https://github.com/user-attachments/assets/02cb0b02-2eee-4642-8241-a9ab3e062917" />


### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Go to the placeholder experiment page: 22124-explat-experiment
- Assign yourself to the `treatment` variation using the bookmarklets
- Visit any of the screens that are currently enforcing the WP Admin view:
  - Posts > All Posts
  - Posts > Tags
  - Posts > Categories
  - Portfolio > All Projects
  - Testimonials > All Testimonials
- Make sure the "View" switcher is not visible
